### ISSUE TITLE
System keybindings blocked by context menus

### ISSUE THREAD

**Issue by pietroalbini** [telegramdesktop/tdesktop#1971]
Telegram Desktop currently overrides system keybindings when context menus are open. This is quite annoying because you're not even able to switch virtual desktop when a context menu is open, forcing you to click away from it.

Any keybinding I tested stopped working, except for `CTRL+ALT+F{1,2,3,4,5,6,7}` which is managed above Xorg.
### Steps to reproduce
1. Right-click on any element of Telegram desktop which supports a context menu (for example a text message in a chat)
2. Press any system keybinding (for example in Unity `Super+W` to open the windows overview)
### Expected behaviour

The action related to that keybind should execute. For example, after pressing `Super+W` an overview of all the open windows should appear.
### Actual behaviour

Nothing happens. Closing the context menu brings the keybindings back.
### Configuration

**Operating system:** Ubuntu 14.04 LTS 64bit  
**Version of Telegram Desktop:** 0.9.42
### Logs

Not needed I think.

**Comment by pietroalbini** [comment]
@john-preston this is still happening in 0.10.1.alpha.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 362 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by pietroalbini** [comment]
This bug still happens in 1.1.19.alpha.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 425 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by pietroalbini** [comment]
This is still happening...

**Comment by ilya-fedin** [comment]
> This is still happening...

Are you sure that this is tdesktop-specific and not Qt-specific?

**Comment by ilya-fedin** [comment]
> Telegram Desktop currently overrides system keybindings when context menus are open

I guess they aren't overriden, just your WM doesn't accept system keybindings when windows with a type of Qt::Popup are open (a generic thing).

**Comment by Aokromes** [comment]
1 month without answer, if this bug is still valid open a new ticket.
